### DISCUSSION TITLE
sprite inside canvas too large in scale

### DISCUSSION THREAD

**Discussion by ronca85** [pixijs/pixijs#7536] [Q&A]
Hello,

I'm using pixi to read a list of images and render a canvas with the same dimensions for each image. There should be a copy of the image inside the canvas and a heavy blur needs to be applied to it. The sprite appears larger than the image - https://codesandbox.io/s/trusting-faraday-voy06?file=/pages/index.vue

A separate issue is the fact that my gpu winds up when I apply blur to all the canvases. Don't know if this is because there are multiple canvases or the blur is just too heavy. If someone has idea why this might be, please let me know.

Thank you

**Comment by ivanpopelyshev** [comment]
Because pixi blur is stupid and big values are handled by doing maaaany passes.
You can make your own blurfilter , that uses blur pass inside by x2 strength, like
`a/1 + a/2 + a/4+ a/8 = s` where s is your resulting strength and `a/8` has to be below the kernel

soo how big is your `strength`?

**Reply by ronca85** [reply]
strength is 10

**Reply by ivanpopelyshev** [reply]
and kernel? try 7, and make less strength
